src/ducks/Signals/signalFormManager/signalCrudForm/signal/TriggerForm.js

```diff
@@ -200,14 +225,24 @@ export const TriggerForm = ({
               name={trigger.title}
               showTrigger={showTrigger}
               showTriggerFunc={showTriggerFunc}
-              actionsEnabled={false} // Make dynamic if trigger more 1 (in future)
+              actionsEnabled={false} // Make dynamic if triggers more than 1 (in future)
             />
 
             {showTrigger && (
               <div className={styles.Trigger}>
                 <div className={styles.row}>
                   <div className={styles.Field}>
-                    <label>Asset</label>
+                    <label>Type</label>
+                    <FormikSelect
+                      name='signalType'
+                      isDisabled={defaultSignalType.isDisabled}
+                      defaultValue={defaultSignalType.value.value}
+                      placeholder='Pick signal type'
+                      options={ASSETS_FILTERS}
+                    />
+                  </div>
+                  <div className={styles.Field}>
+                    <label>&nbsp;</label>
```

Comment:
_(same comment as below)_ Could the same ui effect could be achieved without creating extra node?

Reply:
Will removed